invenio_records_resources/services/records/search.py

```diff
@@ -1,4 +1,13 @@
-"""Example search."""
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Search utilities."""
```

Comment:
Moderate (mergeable if changed). To be consistent with the rest of the `mock_module`.
```suggestion
"""Example search."""
```

In addition @lnielsen we have not set the license header in any file of the `mock_module`. Since it is a mock... Should we?

Reply:
Not in mock_module anymore